### PR TITLE
Use dummy1 instead of lo:100 on Linux

### PR DESCRIPTION
**Pull request by kargig**
A dummy interface is better than using loopback for routing packets to/from.

### CODE REVIEW

**Comment by dguido** [comment]
@kargig Can you address the tests that are failing?

File: roles/common/handlers/main.yml [hunk 1]
```diff
@@ -7,12 +7,18 @@
...
-
 - name: restart loopback bsd
   shell: >
     ifconfig lo100 destroy || true &&
     ifconfig lo100 create &&
     ifconfig lo100 inet {{ local_service_ip }} netmask 255.255.255.255 &&
     ifconfig lo100 inet6 FCAA::1/64; echo $?
+
+- name: load dummy
+  shell: modprobe dummy
```
Comment (gunph1ld) [inline roles/common/handlers/main.yml]: http://docs.ansible.com/ansible/modprobe_module.html

File: roles/common/handlers/main.yml [hunk 2]
```diff
@@ -7,12 +7,18 @@
...
+
+- name: add dummy
+  shell: ip link add dummy100 type dummy
```
Comment (gunph1ld) [inline roles/common/handlers/main.yml]: Need to consider, that dummy will not be UP after reboot

**Comment by kargig** [comment]
I am not sure if I can do anything about travis not being able to load the dummy module since it's run on an LXC container and afaik LXCs can't load kernel modules.

**Comment by gunph1ld** [comment]
Closing due to inactivity

**Pull request closed without merging**